opcua/common/manage_nodes.py

```diff
@@ -85,7 +85,13 @@ def create_variable(parent, *args):
     """
```

Comment:
I don't think that this functionality should be moved to create_variable_type, because this functionality is about creating an variable of a certain type and not about creating a variable type.

This method is used as part of creating variable:
-as part of creating variable for a certain object type 
-as part of creating variable for a object type instance